resources/lib/addonsettings.py

```diff
@@ -650,6 +650,23 @@ def get_release_track():
         release_channel = AddonSettings.store(KODI).get_integer_setting("release_channel")
         return release_channel
 
+    @staticmethod
+    def get_notification_level():
+        """ Retrieves the level for displaying notifications to the end user:
+
+        0: Information.
+        1: Warning.
+        2: Error.
+
+        :return: Array with the levels that we are showing to the end user
+        :rtype: array
```

Comment:
Could you make this `:rtype: list[str]` that way it helps PyCharm a bit better.

Reply:
Sure, I use PyCharm aswell can we also add the .idea folder to the .gitignore ?